core/src/cg/group4/client/query/Data.java

```diff
@@ -0,0 +1,8 @@
+package cg.group4.client.query;
+
+import java.io.Serializable;
+
+/**
+ * Data class that can be extended. Object extending this class can be send to the server.
+ */
+public abstract class Data implements Serializable { }
```

Comment:
Could be removed if you use Serializable instead of this class.. will refactor it later on...